core/remoteviewserver.cpp

```diff
@@ -54,7 +54,9 @@ RemoteViewServer::RemoteViewServer(const QString &name, QObject *parent)
                                                     name), this, "clientConnectedChanged");
 
     m_updateTimer->setSingleShot(true);
-    m_updateTimer->setInterval(10);
+    // Qml render @ ~60fps which is approximatively 16.66ms, not interesting to throttle under that value
+    // because we would probably resend the same frame again.
```

Comment:
That is actually wrong, it does not resend in that case. It also checks if the source actually changed. At least that's what it is supposed to be doing, would be nice to test as well I guess. Ie. if you have a static scene, it should only send you one frame, until you tell it the scene has changed.